app/extensions/string_extensions.rb

```diff
@@ -363,14 +365,17 @@ class String
       "nbsp"  => " "
     }.freeze
   end
+  # rubocop:enable Layout/AlignHash
   # :startdoc:
 
   # This should safely match anything that could possibly be interpreted as
   # an HTML tag.
   HTML_TAG_PATTERN = %r{</*[A-Za-z][^>]*>}.freeze
 
   ### Textile-related methods ###
-  #
+
+  # Disable cop because we need to use html_safe to avoid added escaping
+  # rubocop:disable Rails/OutputSafety
   def t(sanitize = true)
     Textile.textilize_without_paragraph(self, false, sanitize).html_safe
```

Comment:
Seems like it would be better to have a `.textlize_without_paragraph_safe` that takes the same args.  It could then be leveraged by `t` and `tl` and reduce the number of `.html_safe` calls.